xstatic/pkg/font_awesome/__init__.py

```diff
@@ -24,11 +24,14 @@
 KEYWORDS = '%s xstatic' % NAME
 
 # XStatic-* package maintainer:
-MAINTAINER = 'Thomas Waldmann'
-MAINTAINER_EMAIL = 'tw@waldmann-edv.de'
+MAINTAINER = 'Reimar Bauer'
+MAINTAINER_EMAIL = 'rb.proj@gmail.com'
 
 # this refers to the project homepage of the stuff we packaged:
-HOMEPAGE = 'http://fortawesome.github.io/Font-Awesome/'
+HOMEPAGE = 'https://github.com/FortAwesome/Font-Awesome'
+
+# this refers to the location of the xstatic package
+REPOSITORY = 'https://github.com/xstatic-py/xstatic-font-awesome'
```

Comment:
if that is something new, guess we need to add it to the main xstatic pkg also.

Reply:
I just looked for a feature for adding same information into setup.py/pypi as I had by the pyproject.toml.

there I had used
```
[project.urls]
homepage =  
documentation =  
repository = 
```


I have no idea when that was introduced. But I don't want it a random dictionary. The repository of the xstatic package is what we want. At least when there are many projects involved.

https://packaging.python.org/en/latest/guides/distributing-packages-using-setuptools/#project-urls

Reply:
@ThomasWaldmann https://pypi.org/project/XStatic/ there the Home is also the Repository Url. 
We have not an example included but name jquery as the important example so I can better  update jquery.  That is also a good idea, 3.6.3 is the recent one.

Let me know if that is a solution for this.

Reply:
33 needs to say "repository of xstatic-*", not "xstatic package" (that would be pypi).

Reply:
You can squash/merge this afterwards. Github is so slow I won't do further review here.